pkg/azurefile/azure.go

```diff
@@ -102,3 +107,63 @@ func getKubeClient(kubeconfig string) (*kubernetes.Clientset, error) {
 
 	return kubernetes.NewForConfig(config)
 }
+
+func updateSubnetServiceEndpoints(ctx context.Context, az *azure.Cloud, subnetLockMap *lockMap) error {
```

Comment:
since `updateSubnetServiceEndpoints` depends on driver, what about set `func (Driver *d) updateSubnetServiceEndpoints() error`, then you don't need to pass parameters?

Reply:
may fake driver in UT, anyway it's coding style, it's up to you when take this comment.

Reply:
It is an Azure Operation. I think the function is more related to azure than driver. So I wrote it in azure.go with this coding style.